virttest/utils_disk.py

```diff
@@ -807,6 +807,8 @@ def set_drive_letter(session, did, partition_no=1):
         if re.search("Reserved", line, re.I | re.M):
             partition_no += int(line.split()[1])
     assign_letter_cmd = ' echo select partition %s ; echo assign '
+    if target_letter:
+        assign_letter_cmd += 'letter=%s' % target_letter
     assign_letter_cmd = _wrap_windows_cmd(assign_letter_cmd)
     session.cmd(assign_letter_cmd % (did, partition_no))
```

Comment:
```suggestion
    session.cmd(assign_letter_cmd % (did, partition_no))
    if target_letter:
        return target_letter
```
We don't need the following codes to get the letter has been set just return `target_letter` if has set the `target_letter`.